core/simulations/scripts/causeDebt.ts

```diff
@@ -0,0 +1,11 @@
+import hre from 'hardhat';
+import { causeDebt } from '../../helpers/causeDebt';
+import { HARDHAT_PRIVATE_KEY } from '../../helpers/constants';
+import { createWalletForRpc } from '../../helpers/hardhat';
+
+const provider = hre.network.provider;
+
+export default async () => {
+    await createWalletForRpc(HARDHAT_PRIVATE_KEY, 'custom');
```

Comment:
- [ ] Why does `createWalletForRpc` hardhat helper function need to accept `HARDHAT_PRIVATE_KEY` every time it is called? Do we expect other wallets? Then the function wouldn't work anyway, because it doesn't return a wallet, but sets it globally. I think it makes sense to either hardcode `HARDHAT_PRIVATE_KEY` inside this function or at least make it an optional parameter
- [ ] Please always make `network` parameter to be the first argument like everywhere else (you can also make it default for all test helpers since we don't expect them to change)